### ISSUE TITLE
Can I compile Postal's views?

### ISSUE THREAD

**Issue by denisz1** [andrewdavey/postal#92]
It is possible to compile cshtml Razor views.

Can this be done for Postal's views as well? Is there anything I should be aware of, any possible problems, etc.?

**Comment by michaljacks51** [comment]
Absolutely. Compiling Postal's views can provide valuable insights and [find lost children](https://thelost.net/) perspectives. Go ahead and compile them, and feel free to share any interesting findings or analyses.

**Comment by Charlesjack4** [comment]
Sure thing! Compiling Postal's views from episode sounds like a great idea. It could offer valuable insights or highlights from that particular episode. Looking forward to seeing the compilation of [https://sassagrantstatuscheck.co.za/sassa-appeal/](https://sassagrantstatuscheck.co.za/sassa-appeal/).
